app/V1Module/presenters/EmailsPresenter.php

```diff
@@ -78,15 +86,17 @@ public function checkSendToSupervisors() {
    */
   public function actionSendToSupervisors() {
     $supervisors = $this->users->findByRoles(Roles::SUPERVISOR_ROLE, Roles::SUPERADMIN_ROLE);
```

Comment:
This is not a complete list of supervisor roles.

Reply:
True, EmailsPresenter was implemented before role addition and I forgot about that in this reimplementation...

Reply:
The question is how to solve this in more forward-compatible way...

Reply:
I do not think you can (at least not easily). Just update the list.